static/script.js

```diff
@@ -5,7 +5,33 @@ $(function () {
     dateTime.local()
     $element.text(dateTime.format('YYYY-MM-DD HH:mm:ss'))
   })
+
   $('.no-collapsable').on('click', function (event) {
     event.stopImmediatePropagation()
   })
 })
+
+function RefreshMe () { // eslint-disable-line no-unused-vars
+  var RefreshF = localStorage.getItem('refresh_f')
+  $('#auto-refresh-li').removeClass('d-none')
+  window.setInterval(refresh, 30000)
+  if (RefreshF == null) {
+    localStorage.setItem('refresh_f', 'is_off')
+  }
+  if (RefreshF === 'is_on' && !$('#auto-refresh').is(':checked')) {
+    $('#auto-refresh').click()
+  }
+  $('#auto-refresh').on('change', function (event) {
+    if (RefreshF === 'is_on') {
```

Comment:
You never updating ``RefreshF``, so only first change of button's state is storing in localStorage (try click two times, refresh page and see in which state is button). Updating RefreshF on any change or don't store this in local variable.